tuf/client/client.go

```diff
@@ -422,7 +420,8 @@ func (c *Client) downloadTargets(role string) error {
 }
 
 func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte) ([]byte, *data.Signed, error) {
-	raw, err := c.remote.GetMeta(role, size)
+	rolePath := utils.URLFilePath(role, expectedSha256)
```

Comment:
It's not a perfect name and it doesn't even construct a full path, it literally just puts the checksum and role name together. The RemoteStore deals with adding any appropriate file extensions and path prefix (i.e. /v2/<gun>/_trust/)